src/redrose-launcher/main.sh

```
@@ -0,0 +1,82 @@
+tools=(curl jq wget qemu-system-x86_64 qemu-img)
+MISSING=0
+home=$(realpath ~)
+
+drive_path=~/.cache/redrose.qcow2
+
+download_url () {
+    iso_path="${home}/.cache/redrose-$2.iso"
+
+    echo -e "${GREEN}[!] Downloading : $1${NC}"
+    if [ ! -f "$iso_path" ]; then
+        wget -q --show-progress -O ${iso_path} $1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)